DynamicAnalyzer/views/android/environment.py

```diff
@@ -420,4 +455,6 @@ def start_frida():
         trd = threading.Thread(target=start_frida)
         trd.daemon = True
         trd.start()
-        logger.info('Frida Server is running')
+        logger.info('Frida Server is going to start')
```

Comment:
Can you reword this to `Starting Frida Server`

Reply:
Changed it, please review again.